devicon.json

```
@@ -10455,6 +10455,25 @@
             }
         ]
     },
+    {
+        "name": "web3js",
+        "altnames": [],
+        "tags": [
+            "blockchain",
+            "ecommerce"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#ffffff",
```

This seems to be the main color of the logo:
```suggestion
        "color": "#bcbec0",
```

---

I know it's not the main color of the logo, but on their site they use `#f16822` as an accent color and it's present on the styled "3" in the logo.

---

@lunatic-fox fair. I'll change the suggestion to that color